src/fixit/config.py

```diff
@@ -156,26 +224,82 @@ def merge_configs(
 
     Assumes raw_configs are given in order from highest to lowest priority.
     """
-    kwargs: Dict[str, Any] = {
-        "root": None,
-    }
+
+    enable_rules: Set[str] = set()
+    disable_rules: Set[str] = set()
+    local_paths: List[Path] = []
+    rule_options: RuleConfigs = {}
+
+    def process_subpath(
+        subpath: Path,
+        *,
+        enable: Sequence[str] = (),
+        disable: Sequence[str] = (),
+        options: RuleConfigs = frozendict(),
+    ):
+        subpath = subpath.resolve()
+        if not path.is_relative_to(subpath):
+            print(f"ignoring {subpath}")
+            return
+
+        for rule in enable:
+            if rule.startswith("."):
```

Comment:
IIUC, this hints at local rules being distinguished from packaged ones by a `.` prefix? I like this very much, wanna add it to [the proposal](https://github.com/amyreese/Fixit/tree/config-prototype)?

Reply:
We should also maybe attach `local_path` to the `rule` to avoid collisions?
```
root/.fixit.toml  # has enable = [".foo"]
root/foo.py
root/subdir/.fixit.toml  # has enable = [".foo"]
root/subdir/foo.py
```

Reply:
I agree that this is needed, but I'm going to leave this as-is for the moment, and think about how to do it right.